receiver/apachesparkreceiver/metadata.yaml

```diff
@@ -4,4 +4,783 @@ status:
   class: receiver
   stability: 
     development: [metrics]
-  distributions: [contrib]
\ No newline at end of file
+  distributions: [contrib]
+
+attributes:
+  application_id:
+    description: The ID of the application for which the metric was recorded.
+    type: string
+  application_name:
+    description: The name of the application for which the metric was recorded.
+    type: string
+  stage_id:
+    description: The ID of the application stage for which the metric was recorded.
+    type: int
+  attempt_id:
+    description: The ID of the stage attempt for which the metric was recorded.
+    type: int
+  stage_status:
+    description: The status of the stage for which the metric was recorded.
+    type: string
+    enum:
+      - ACTIVE
+      - COMPLETE
+      - PENDING
+      - FAILED
```

Comment:
We typically don't report status information in attributes. My recommendation would be to model this as a metric with one hot encoded attributes. See https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6892#issuecomment-1007446915